tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -540,3 +544,50 @@ def test_vmgenid(guest_kernel_linux_6_1, rootfs, microvm_factory, snapshot_type)
 
         # Update the base for next iteration
         base_snapshot = snapshot
+
+
+@pytest.mark.skipif(
+    platform.machine() != "aarch64"
+    or not global_props.is_ec2
```

Comment:
why does the test depend on ec2? do you want to check for AL2023?

Reply:
Here I assume that if CI is run on EC2, it will be either amazon linux or ubuntu from our pipelines and those will have a kernel with register resetting capability. The check is not perfect though. Someone can use EC2 instance with non patched kernel, but it is a low probability and does not cause much harm.

Reply:
Removed the `is_ec2` check for now. Will need to add it back when AL kernels have the patch

Reply:
You can check that is AmazonLinux with `global_props.host_os == "amzn2"`